src/util/path.cpp

```diff
@@ -221,7 +221,7 @@ std::vector<std::string> read_dir(std::string const &dirname) {
 }
 
 std::string lrealpath(std::string const & fname) {
-#if defined(LEAN_EMSCRIPTEN)
+#if defined(LEAN_EMSCRIPTEN_BROWSER)
     return fname;
```

Comment:
Does something like the following work instead of compiling `path.cpp` twice?
```c++
#if defined(LEAN_EMSCRIPTEN)
if (EM_ASM_INT(typeof window === "object")) {
  return fname;
}
```

Reply:
Yeah, that's great! I should be able to revert a lot of the CMake complications I had to add with this.